kerchunk/xarray_backend.py

```diff
@@ -48,6 +48,12 @@ def open_reference_dataset(
     if open_dataset_options is None:
         open_dataset_options = {}
 
+    if open_dataset_options.get("consolidated"):
+        raise ValueError(
```

Comment:
A warning would do? Without making this check, it only results in a warning from the zarr engine anyway I think. Note that parquet references do work as consolidated or not, it doesn't matter.

Reply:
Ah good to know! Happy to close the PR.